journal/updated/01-journal.adoc

```diff
@@ -1031,237 +1031,250 @@ and the high-sheriff continued loving and kind to Friends,
-He was very loving and kind to us, and let Friends go themselves to Bridgenorth,
-about fourteen miles, where the court sessions were then held;
-and he desired me to stay with him in his affliction, and not be much from him.
+He was very loving and kind to us, and let Friends go alone to Bridgenorth,
+a distance of about fourteen miles, where the court sessions were then held;
+and he desired me to stay with him in his affliction, and not be much absent from him.
 He said his life was at stake, and if God and the judge would show him any mercy,
-it was upon our Friends`' account, and not for any deserts that were in himself,
-for he confessed he had been too severe to us; but notwithstanding, said he,
-you are merciful men, and can forgive wrongs and injuries.
+it was upon our Friends`' account, and not for any goodness that was in himself,
+for he confessed he had been too severe to us; "`but notwithstanding,`" said he,
+"`you are merciful men, and can forgive wrongs and injuries.`"
 
 When we came to Bridgenorth,
-we were put in a large spacious room in the house of correction,
-to be there in the day time, that we might be all together, and ready when called for;
-but we had liberty of going in and out for lodging, and what necessaries we needed;
-no keeper being over us, but what we set ourselves to look to the door,
-and that too many Friends might not be out at once,
-and those were not to stay out too long.
-We saw it was convenient, that Friends should go out by two and two, to walk the streets;
+we were put in a large, spacious room in the house of correction.
+We arrived there in the day time, so that we might all be together and ready when called for;
+but we had the liberty to go in and out for lodging and what necessaries we needed,
+no keeper being over us but those which we set ourselves to watch the door,
+to ensure that too many Friends might not be out at once,
+and that those who were out would not be out too long.
+We saw it was safer that Friends go out in groups of two to walk the streets;
 for it was a strange thing to people to hear of Quakers.
 
-Once it fell to my lot to be at the door, though the door was always open,
-that such who would,
-might come and see us--with several of whom we had reasonings and
-disputes about the way of truth and righteousness--there came one,
-who appeared something like a gentleman, and asked me, whether he might see the Quakers;
-I desired him to walk up along with me, and he should see them.
-When I had brought him up to the room where Friends were walking, I told him,
-those were they.
-He answered, these be Christians like ourselves, but where are the Quakers?
-I told him, these were they that were called Quakers.
-He asked me, whether I was one of them; I told him, I was one so called.
+Once it fell to my lot to be at the door. The door was always open,
+so that those who desired to do so
+might come and see us (several of whom we had reasonings and
```

Comment:
this needs the "with".  (**with** several of whom

Reply:
It does have the with, just later in the sentence. It sounded better to me that way, but I can change it if you want.